### ISSUE TITLE
services.db must be a mapping

### ISSUE THREAD

**Issue by sanjay-thiyagarajan** [CanastaWiki/Canasta-DockerCompose#13]
When I tried to run ```docker-compose up -d``` , I'm getting this error.
OS - Windows 11
Branch - ```main```  

![image](https://user-images.githubusercontent.com/42594454/166493152-08623e54-89c9-484a-b03e-1a5683f30ca9.png)  

![image](https://user-images.githubusercontent.com/42594454/166493242-9876fd06-46fd-42b1-b7a9-6780b5756860.png)

**Comment by jeffw16** [comment]
1. Did you modify docker-compose.yml?
2. What are the contents of your docker-compose.override.yml?

**Comment by sanjay-thiyagarajan** [comment]
I didn't modify any of those files. It's just the same as the current version of ```main``` branch

**Comment by jeffw16** [comment]
We are getting this issue at MyWikis on a Mac as well, so reopening now.

**Comment by amalpaul54111** [comment]
This seems to be an issue regarding the ```docker-compose.override.yml``` as removing that file does solve the problem.

**Comment by amalpaul54111** [comment]
Maybe we don't need to mention all the services in the ```docker-compse.override.yaml```, the user can mention each one as it's required. That would solve the issue.

```
services:
  web:
    image: ghcr.io/canastawiki/canasta:1.0.1
```

**Comment by jeffw16** [comment]
Are you sure? I have `docker compose up -d` working with a clean clone from this repo, keeping `docker-compose.override.yml` without making any changes.

**Comment by amalpaul54111** [comment]
I could not get it working from a clean clone of this repo, keeping the ``` docker-compose.override.yml```. But I will need to try it on some other system to verify that.

**Comment by jeffw16** [comment]
This is very strange.

**Comment by jeffw16** [comment]
Removing the other services did work for @rpunjani so I'm going to submit a PR to comment those out of the file. (We should keep them in there for reference, in case someone wants to add changes to any of these containers.)

**Comment by olsonjaredm** [comment]
Confirmed we saw the same issue on 1.0.1, commenting the unused ones worked, and the issue is resolved in 1.1.0.
